landing-page/content/common/quickstarts.md

```diff
@@ -0,0 +1,335 @@
+---
+url: quickstarts
+toc: true
+aliases:
+    - "quickstarts"
+    - "getting-started"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+## <img src="../img/spark-menu-logo.png"> Spark and Iceberg Quickstart
```

Comment:
I'd probably not include the Spark logo here. I think logos in nav menus can make sense, but I don't think they add value for page content. If you're adding a section with buttons to select the quickstart, that's a good idea for the logo (larger target to click on) but I think we should hold off until there's a selection.

Reply:
Removed 👍